Ginger/GingerCoreCommon/WorkSpaceLib/Solution.cs

```diff
@@ -78,6 +79,27 @@ public string Name
         [IsSerializedForLocalRepository]
         public ObservableList<RepositoryItemTag> Tags;
 
+
+        private AccessibilityConfiguration mDefaultExcludeRule;
```

Comment:
Remove accessibility rules reference from solution.cs as it is not being used

Reply:
Removed